doc/source/whatsnew/v0.20.0.txt

```diff
@@ -580,3 +580,4 @@ Bug Fixes
 - Bug in ``Series.replace`` and ``DataFrame.replace`` which failed on empty replacement dicts (:issue:`15289`)
 - Bug in ``pd.melt()`` where passing a tuple value for ``value_vars`` caused a ``TypeError`` (:issue:`15348`)
 - Bug in ``.eval()`` which caused multiline evals to fail with local variables not on the first line (:issue:`15342`)
+- Bug in ``.read_csv()`` which caused ``parse_dates={'datetime': [0, 1]}`` to fail with multiline headers (:issue:`15376`)
```

Comment:
don't put this as the last line, instead use an empty space, otherwise you will get conflicts.

``Bug in .read_csv() where parse_dates with a list-of-integers specified would fail with multiline headers``